types/unit/service.pp

```diff
@@ -93,14 +93,38 @@
     Optional['OOMPolicy']                 => Enum['continue', 'stop','kill'],
     Optional['OOMScoreAdjust']            => Integer[-1000,1000],
     Optional['Environment']               => String,
-    Optional['EnvironmentFile']           => Variant[
-      Stdlib::Unixpath,Pattern[/-\/.+/],
-      Array[Variant[Stdlib::Unixpath,Pattern[/-\/.+/]],1],
-    ],
+    Optional['EnvironmentFile']           => Variant[Stdlib::Unixpath,Pattern[/-\/.+/], Array[Variant[Stdlib::Unixpath,Pattern[/-\/.+/]],1]],
```

Comment:
Why this change? Other parameter seems to use the same data type (`BindPaths`, `BindReadOnlyPaths`, maybe others), maybe it make sense to add another custom type for this?

Reply:
I reverted the change